tools/flang2/flang2exe/llassem.cpp

```diff
@@ -1168,11 +1168,16 @@ assemble_end(void)
       get_typedef_ag(gname, tname);
       tdefsym = find_ag(gname);
       typed = AG_TYPENAME(tdefsym);
-      fprintf(ASMFIL, "%%struct%s = type < { %s } > \n", name, typed);
-      fprintf(ASMFIL, "@%s = %s global %%struct%s ", name,
-              AG_ISMOD(gblsym) ? "external" : "common", name);
-      fprintf(ASMFIL, "%s, align %d",
-              AG_ISMOD(gblsym) ? "" : " zeroinitializer", align_value);
+      if (AG_CMBLKINITDATA(gblsym)) {
+        fputs (AG_CMBLKINITDATA(gblsym), ASMFIL);
+        free (AG_CMBLKINITDATA(gblsym));
+      } else {
+        fprintf(ASMFIL, "%%struct%s = type < { %s } > \n", name, typed);
```

Comment:
Can we have a test for the else part also?

Reply:
those lines are existing code, just moved to else part because of added if cond.